pymeasure/instruments/tti/thurlby1905a.py

```diff
@@ -0,0 +1,110 @@
+#
+# This file is part of the PyMeasure package.
+#
+# Copyright (c) 2013-2025 PyMeasure Developers
+#
+# Permission is hereby granted, free of charge, to any person obtaining a copy
+# of this software and associated documentation files (the "Software"), to deal
+# in the Software without restriction, including without limitation the rights
+# to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+# copies of the Software, and to permit persons to whom the Software is
+# furnished to do so, subject to the following conditions:
+#
+# The above copyright notice and this permission notice shall be included in
+# all copies or substantial portions of the Software.
+#
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+# IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+# FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+# AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+# LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+# OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+# THE SOFTWARE.
+#
+
+import logging
+
+from pymeasure.errors import Error, RangeError
+from pymeasure.instruments import Instrument
+
+log = logging.getLogger(__name__)
+log.addHandler(logging.NullHandler())
+
+
+class Thurlby1905a(Instrument):
+    """Represents the Thurlby 1905a intelligent digital multimeter
+
+    .. code-block python
+
+        from pymeasure.instruments.tti import Thurlby1905a
+
+        dmm = Thurlby1905a("ASRL/dev/ttyACM0::INSTR", baud_rate=2400)
+        output = dmm.read_measurement
+
+    """
+
+    def __init__(self, adapter, name="Thurlby 1905a", **kwargs):
+        kwargs.setdefault("read_termination", "\r")
```

Comment:
Can you change the read_termination? If not, just write it into the super.init call.

Reply:
Can guarantee that `read_termination`, for this version at least, is '\r'. Will do as suggested.